src/soep_preparation/clean_modules/bioparen.py

```diff
@@ -19,16 +19,23 @@ def clean(raw_data: pd.DataFrame) -> pd.DataFrame:
     """
     out = pd.DataFrame()
     out["p_id"] = apply_smallest_int_dtype(raw_data["pid"])
+    out["hh_id_original"] = apply_smallest_int_dtype(raw_data["cid"])
 
-    # Social father personal id
-    out["p_id_father"] = float_to_int(
-        series=raw_data["fnr"],
+    out["p_id_father_1"] = float_to_int(
+        series=raw_data["fnr1"],
+        drop_missing=True,
```

Comment:
This should be a good occasion to tackle this. IIUC, `drop_missing` is not at all what happens; rather, negative values are replaced by missing values. Is that correct? If so, rename the keyword to `code_negative_values_as_na`. Remove its default value, better be explicit everywhere.